src/aks-preview/setup.py

```diff
@@ -9,7 +9,7 @@
 
 from setuptools import setup, find_packages
 
-VERSION = "7.0.0b8"
+VERSION = "8.0.0b1"
 
```

Comment:
if stable version is expected, you can publish a version like 7.0.0.
The order of semantic versioning is: 7.0.0b1 < 7.0.0b2 < ...< 7.0.0b8 < 7.0.0 < 7.0.1 < ...< 8.0.0b1 < 8.0.0b2... < 8.0.0
`b` represents `pre` version of stable release version.

Reply:
We will keep it as a preview extension for the foreseeable future, which means that the AKS team will only provide service support for this extension in a best effort manner.

In this case, I suppose 8.0.0b1 is a good choice as we're making breaking change (removing several deprecated options), right?

Reply:
according to extension versioning rule here: https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md#notes-1

"The major version number of preview can increase by at most one compared to the last stable version. For example, the last stable version is 2.0.0, if there are breaking changes for next preview release, the version will be 3.0.0b1. Then another breaking changes for the next preview release, the version will be 3.0.0b2"

It seems aks-preview does not process any stable versions, so major tag added for this release.

Reply:
Please plan module's a major version bump with a foresight manner, referred to the semantic doc here:  https://semver.org/#if-even-the-tiniest-backward-incompatible-changes-to-the-public-api-require-a-major-version-bump-wont-i-end-up-at-version-4200-very-rapidly

Reply:
Thanks for the reminder. We will not make breaking changes unless necessary, but the preview API and cli-extensions provided by AKS are intended to provide preview features without SLA.